lib/GeometryMath.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+
+module.exports = {
+    sumBarycentric: sumBarycentric,
+    flattenTriangle: flattenTriangle,
+    pointLineDistanceParametric: pointLineDistanceParametric
+};
+
+// Functions for working with geometry like triangles and lines
+
+var sumBarycentricScratch = new Cartesian3();
+function sumBarycentric(barycentric, vector0, vector1, vector2, result) {
```

Comment:
We don't use pragmas or optimize out `DeveloperError` in node code and I doubt we will anytime soon.  The pragma is specifically an AMD/requirejs thing so it doesn't exist in Node and even if we decided to do some sort of Node build, server side code needs more robust logging than client code; so we probably would keep them anyway.  So definitely check for and throw `DeveloperError`, but you can leave out the pragma.